routing/transit_graph.cpp

```diff
@@ -198,13 +362,29 @@ transit::Gate const & TransitGraph::GetGate(Segment const & segment) const
   return it->second;
 }
 
+::transit::experimental::Gate const & TransitGraph::GetGatePT(Segment const & segment) const
+{
+  CHECK_EQUAL(m_transitVersion, ::transit::TransitVersion::AllPublicTransport, ());
+  auto const it = m_segmentToGatePT.find(segment);
+  CHECK(it != m_segmentToGatePT.cend(), ("Unknown transit segment."));
+  return it->second;
+}
+
 transit::Edge const & TransitGraph::GetEdge(Segment const & segment) const
 {
   auto const it = m_segmentToEdge.find(segment);
   CHECK(it != m_segmentToEdge.cend(), ("Unknown transit segment."));
   return it->second;
 }
 
+::transit::experimental::Edge const & TransitGraph::GetEdgePT(Segment const & segment) const
+{
+  CHECK_EQUAL(m_transitVersion, ::transit::TransitVersion::AllPublicTransport, ());
+  auto const it = m_segmentToEdgePT.find(segment);
+  CHECK(it != m_segmentToEdgePT.cend(), ("Unknown transit segment."));
```

Comment:
`CHECK(it != m_segmentToEdgePT.cend(), ("Unknown transit segment.", segment));`?